frontend/cypress/tests/chartView/chartView_errorSummary_messageBox.cy.js

```diff
@@ -27,4 +27,24 @@ describe('error summary', () => {
     cy.get('.error-message-box')
       .should('not.be.visible');
   });
+
+  it('number of error messages', () => {
```

Comment:
We could make this test case name more descriptive
```suggestion
  it('can be expanded and collapsed if count > 4', () => {
```